src/timely-util/Cargo.toml

```diff
@@ -11,6 +11,8 @@ workspace = true
 
 [dependencies]
 differential-dataflow = "0.12.0"
+either = "1"
```

Comment:
Do we actually use either? Can't see it on mobile!

Reply:
Yes, the stacked wrapper needs to define an iterator associated type which either the one from timelystack or the one from chunkedstack